### ISSUE TITLE
Quote next character doesn't work for Ctrl-L

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#687]
_[Trac time 20090608 224559Z]_ C-q C-l should insert the C-l character but instead raises an error:

```
Traceback (most recent call last):
  File "/home/rob/src/peppy-multikey/peppy/lib/multikey.py", line 1300, in OnMenu
    keystroke = self.findKeystrokeFromMenuId(eid)
  File "/home/rob/src/peppy-multikey/peppy/lib/multikey.py", line 1319, in findKeystrokeFromMenuId
    return self.findKeystrokeFromMenuAction(action)
  File "/home/rob/src/peppy-multikey/peppy/lib/multikey.py", line 1339, in findKeystrokeFromMenuAction
    raise KeyError
KeyError
```

I think it's because C-l is bound to a specific window rather than a global action?

**Comment by robmcmullen** [comment]
_[Trac time 20090608 225548Z]_ Nope, that's not it.  New guess: I bet it's caused by duplicate keystrokes that aren't being flagged.

**Comment by robmcmullen** [comment]
_[Trac time 20090608 230244Z]_ Yep, that's the problem.  C-l is also mapped to the Scintilla action CutLine, but C-l shows up in the menu as RecenterScreen.  The keybiding code doesn't catch that there's a duplicate.

**Comment by robmcmullen** [comment]
_[Trac time 20090608 231914Z]_ [old svn rev2113] Fixed #687: check for precedence in duplicate keys
- if a keystroke is bound multiple times, the one that appears in a menu item takes priority
- if none are in menus, the last one bound takes precedence
